docs/submarine-sdk/pysubmarine/tracking.md

```diff
@@ -0,0 +1,63 @@
+<!---  
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+   http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  
+  See the License for the specific language governing permissions and 
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# pysubmarine-tracking
+- Allow data scientist to track distributed ML job
+- Support store ML parameters and metrics in Submarine-server
+- Support store ML job output (e.g. csv,images)
+- Support hdfs,S3 and mysql
+- (WEB) Metric tracking ui in workbench-web
+- (WEB) Metric graphical display in workbench-web
+
+## Functions
+### `submarine.get_tracking_uri()`
+
+return the tracking URI.
+
+### `submarine.set_tracking_uri(URI)`
+
+set the tracking URI. You can also set the
+SUBMARINE_TRACKING_URI environment variable to have Submarine find a URI from
+there. The URI should be database connection string. 
+
+**Parameters**
+
+- URI - Submarine record data to a SQLAlchemy compatible database. The database URL
```

Comment:
Submarine is a huge project and there are too many places to develop.
Support for multiple database types looks good, But it will consume a lot of our time.
We are very good at supporting a database very well.
So submarine only supports mysql database,
My suggestion is that pysubmarine only needs to support mysql.